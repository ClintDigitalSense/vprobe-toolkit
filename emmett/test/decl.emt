VMM1Hz {
   strvar = "myStr";
   if (strcmp(strvar, "foo")) {
      printf("garbanzo: %d\n", 666);
      printf("mangofondue: %s\n", strvar);
   }
}
