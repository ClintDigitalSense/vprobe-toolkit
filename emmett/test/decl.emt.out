(version 0.2)
(defstring strvar)

 ;;;;;
(vprobe VMM1Hz ( do  ( setstr strvar   "myStr")( cond  ( (strcmp strvar  "foo" )( do  ( printf    "garbanzo: %d\n"  666 )( printf    "mangofondue: %s\n"  strvar ))))))